src/components/ClientList/index.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import PersonCardList from "../PersonCardList";
+import "./style.css";
+
+const ClientList = ({ data }) => {
```

Comment:
Do you really need this component?
Maybe you realize this all functionality in one `PersonCardList` component?